config/hubs/schema.yaml

```diff
@@ -179,6 +179,7 @@ properties:
                   - google-oauth2
                   - github
                   - password
+                  - CILogon
```

Comment:
I was expecting all lowercase here, but it is not a big deal at all 😉.

Reply:
Ah wow! Yes, lowercase would have been better. Will keep this in mind and when I'll document this, I will also change the casing :sweat_smile: TY!